### ISSUE TITLE
.funcignore is ignored

### ISSUE THREAD

**Issue by JohnLBevan** [microsoft/vscode-azurefunctions#3885]
`.funcignore` is expected to behave like a `.gitignore` file, only preventing files from being published to an Azure Function, rather than preventing them from being committed.

For example, with a project structure such as this:
```
.
|- .funcignore
|- .gitignore
|- host.json
|- profile.ps1
|- requirements.psd1
|- ExampleFunction1\
  |- function.json
  |- run.ps1
  |- confidential.secret
```

Where `.funcignore` contains this:

```
*.secret
```

After using VSCode to deploy the function, in the `Code & Test` window we see `MyFunctionApp\ExampleFunction1\confidential.secret` exists in the app's directory.

The same issue is mentioned here on the related Azure Functions Github Action thread: https://github.com/Azure/functions-action/issues/36#issuecomment-1767784868

**Comment by alexweininger** [comment]
Similar to https://github.com/microsoft/vscode-azurefunctions/issues/3882

**Comment by alexweininger** [comment]
Hi @JohnLBevan, a fix for this has been released in [v1.13.2](https://github.com/microsoft/vscode-azurefunctions/releases/tag/v1.13.2) of the extension.
